src/components/content/conditions/helpers/columns.tsx

```diff
@@ -34,8 +37,59 @@ export const patientConditionsColumns: TableColumn<ConditionModel>[] = [
         </div>
       </div>
     ),
+  },
+  {
+    title: "Details",
+    widthPercent: 40,
+    minWidth: 132,
+    render: (condition) => {
+      const onsetText = condition.isSummaryResource ? "Earliest known onset:" : "Onset:";
+
+      return (
+        <div className="ctw-pc-onset-notes">
+          {condition.onset && (
+            <div>
+              {onsetText} {condition.onset}
+            </div>
+          )}
+          <div className="ctw-pc-notes">{condition.notes.join(" ")}</div>
+        </div>
+      );
+    },
+  },
+];
+
+export const patientConditionsAllColumns = (builderId: string): TableColumn<ConditionModel>[] => [
+  {
+    title: "Name",
+    widthPercent: 30,
+    minWidth: 320,
+    render: (condition) => (
+      <div>
+        <div className="ctw-flow-root group-hover:ctw-underline">
+          {capitalize(condition.display)}
+          <span className="ctw-float-right">
+            {condition.ownedByBuilder(builderId) ? (
+              <FontAwesomeIcon className="ctw-text-content-light" icon={faCircleCheck} />
+            ) : (
+              <></>
+            )}
+          </span>
```

Comment:
Can use `null` instead of a document fragment. Can probably omit rendering that empty span as well unless it's needed for some layout reason
```suggestion
          {condition.ownedByBuilder(builderId) ? (
            <span className="ctw-float-right">
              <FontAwesomeIcon className="ctw-text-content-light" icon={faCircleCheck} />
            </span>) : null}
```